datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonJob.scala

```diff
@@ -25,12 +25,16 @@ import org.apache.spark.sql.{Column, DataFrame, SparkSession}
 import za.co.absa.hermes.utils.HelperFunctions
 
 object DatasetComparisonJob {
+  case class GenericPair[+T](reference: T, actual: T)
```

Comment:
Why `GenericPair`? Generic pair is `Pair`, right? If a class has 'Pair' in the name it might be expected to inherit from `Pair`. A more specific name might fit this better.

Reply:
ComparisonPair it is